docs/topics/api/addons.rst

```diff
@@ -60,15 +60,17 @@ This endpoint allows you to search through public add-ons.
     ==============  ==========================================================
          Parameter  Description
     ==============  ==========================================================
-           created  Creation date, descending
-         downloads  Number of weekly downloads, descending
+           created  Creation date, descending.
+         downloads  Number of weekly downloads, descending.
            hotness  Hotness (average number of users progression), descending.
             rating  Bayesian rating, descending.
-           updated  Last updated date, descending
+         relevance  Search query relevance, descending.
```

Comment:
With readthedocs styling we're constrained by the width of the body - if we add too much stuff the table will be horizontally scrollable instead of wrapping the contents, so adding too much stuff to the columns isn't ideal.